### ISSUE TITLE
Reduce the size of the onboarding modal

### ISSUE THREAD

**Issue by qaid** [qaid/look-ma-no-hands#63]
Reduce the size of the onboarding modal because in most instances when the permissions flow begins, the macOS accessibility, privacy & security alert is actually obscured by the onboarding modal and the user cannot see the permissions and therefore cannot set up the app.
<img width="2004" height="1476" alt="Image" src="https://github.com/user-attachments/assets/38c52564-e728-4758-be6a-ece470ba16a4" />
